script.py

```diff
@@ -92,6 +104,17 @@ def is_valid_bus(user_input, response_data_JSON):
     return False
 
 
+def available_buses():
+    request = request_vehicles(token)
+
+    if len(request.json()['data']) == 0:
```

Comment:
You can just use `if not request.json()['data']:` to improve it.

Ref: https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty